.changeset/cuddly-crabs-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+"@definitelytyped/header-parser": minor
+"@definitelytyped/dtslint": minor
```

Comment:
There must be something wrong with changesets after master -> main; this should include all modified packages.

Reply:
I manually excluded packages that only have comment changes. Was that not correct?

Reply:
Ah, I see! That's fine.